pkg/operator/observe_config_test.go

```diff
@@ -39,3 +42,48 @@ func TestObserveClusterConfig(t *testing.T) {
 		t.Error(restrictedCIDRs[1])
 	}
 }
+
+type FakeImageConfigLister struct {
```

Comment:
Creative.  I owed you 
```go
indexer := cache.NewIndexer(controller.KeyFunc, cache.Indexers{})
for _, obj := range test.startingClusterRoles {
	indexer.Add(obj)
}
rbaclisters.NewClusterRoleLister(indexer)
```

So I'll take it, but I think this construction is easier